src/org/omegat/filters2/master/PluginUtils.java

```diff
@@ -114,6 +115,11 @@ public static void loadPlugins(final Map<String, String> params) {
                         // found main manifest - not in development mode
                         foundMain = true;
                     }
+                    if ("theme".equals(m.getMainAttributes().getValue("Plugin-Category"))) {
+                        String target = mu.toString();
+                        Arrays.stream(urls).filter(url -> target.contains(url.toString()))
+                                .forEach(THEME_PLUGIN_JARS::add);
```

Comment:
I appreciate the aesthetics of this style, but I should point out that I think this is much less performant than a simple `for` loop with an `if` statement.

Reply:
done